tests/meson.build

```diff
@@ -0,0 +1,147 @@
+# FIXME: installed-tests is missing here
+
+env_tests = environment()
+env_tests.set('XDP_UNINSTALLED', '1')
+env_tests.set('XDG_DATA_DIRS', meson.current_build_dir() / 'tests' / 'share')
+# FIXME: automake updates PATH, don't think that's necessary here, double-check
+
+subdir('dbs')
+subdir('portals')
+subdir('services')
+
+test_db = executable('testdb',
+                    ['testdb.c'] + src_db,
+                    dependencies: [deps_common],
+                    include_directories: [inc_builddir])
+# This relies on dbs/meson.build to complete first which I think is always
+# true. Can't figure out how to add a configure_file() output as dependency
+# for the test.
```

Comment:
Meson doesn't guarantee that directory A completes before directory B, so if `dbs/` needs to be *built* first, then there needs to be an explicit dependency.

If the file only needs to exist *before running the test* then I think that's automatic.

Reply:
that's the case here, I'll amend the comment

Reply:
> Can't figure out how to add a configure_file() output as dependency

configure_file is, by definition, created at configure time. No build rules are generated for it. It can never fail to exist on time.

Reply:
:facepalm:

yep, that'd explain it. removed the comment, thanks